src/api-reference/budget/v4.budget-header.md

```diff
@@ -817,6 +1255,111 @@ Authorization: Bearer {token}
   }
 ```
 
+#### Team Budget Example
+
+Provide syncGuid when doing an update call. Also as of now we are not supporting approvers, managers and tracking field options for Team Budgets. 
+So you do not have to pass values for those fields.
+
+```json
+  {
+      "name": "TestTeamBudget",
+      "isTest": false,
+      "budgetType": "TEAM",
+      "budgetItemStatusType": "OPEN",
+      "description": "TestTeamBudget",
+      "syncGuid": null,
+      "costObjects": [],
+      "periodType": "YEARLY",
+      "active": true,
+      "currencyCode": "USD",
+      "annualBudget": 1550.00000000,
+      "budgetCategory": null,
+      "owner": {
+          "externalUserSyncGuid": "8000011190000003453",
+          "employeeUuid": "F06D36B8AA9046358B8778D3B2A14FCF",
+          "email": "meenapriya.meenakshisundaram@sap.com",
+          "externalUserCUUID": "8000011190000003453"
+      },
+      "budgetManagers": [],
```

Comment:
If no values should be passed for these fields, then I recommend to change the example to remove the fields from the payload assuming the API works that way.

Reply:
Changes done